app/containers/MessageBox/ReplyPreview.js

```diff
@@ -63,21 +77,54 @@ export default class ReplyPreview extends Component {
 	close = () => {
 		const { close } = this.props;
 		close();
-	}
+	};
+
+	renderImage = () => {
```

Comment:
This will fix issue for only image , but there could be other attachments too in reply  and there is no way to render them , can you extend your PR to render those things to , it will go something like this https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/fd5fbe47e8e067e3e7af05d1780c404ca62426b0/app/containers/message/Message.js#L232

Reply:
Sure. I will update the code to render all the attachments.